packages/warehouses/src/warehouseClientFromCredentials.ts

```diff
@@ -1,16 +1,17 @@
 import {
     CreateWarehouseCredentials,
-    UnexpectedServerError,
     WarehouseTypes,
+    UnexpectedServerError,
 } from 'common';
-import Logger from '../../logger';
-import { WarehouseClient } from '../../types';
-import BigqueryWarehouseClient from './BigqueryWarehouseClient';
-import DatabricksWarehouseClient from './DatabricksWarehouseClient';
-import PostgresWarehouseClient from './PostgresWarehouseClient';
-import RedshiftWarehouseClient from './RedshiftWarehouseClient';
-import SnowflakeWarehouseClient from './SnowflakeWarehouseClient';
+import * as Logger from 'winston';
+import { WarehouseClient } from './types';
+import { SnowflakeWarehouseClient } from './warehouseClients/SnowflakeWarehouseClient';
+import { BigqueryWarehouseClient } from './warehouseClients/BigqueryWarehouseClient';
+import { DatabricksWarehouseClient } from './warehouseClients/DatabricksWarehouseClient';
+import { PostgresWarehouseClient } from './warehouseClients/PostgresWarehouseClient';
+import { RedshiftWarehouseClient } from './warehouseClients/RedshiftWarehouseClient';
 
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Should we disable this rule in eslint.tc ?

Reply:
I'm not sure tbh! There was a mixture of approaches with the adapters so I just did this as standard